src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewController.swift

```diff
@@ -33,6 +33,9 @@ class EUSettingsViewController: DynamicTableViewController {
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
+		if #available(iOS 13, *) {
```

Comment:
is the `if #available(iOS 13, *)` really needed?

Reply:
@marcussc thanks for your review, and especially for mentioning this one.

Yes, I think this line of iOS 12 code https://github.com/corona-warn-app/cwa-app-ios/blob/d79316e5804952e0734f73036d97dc55e8516166/src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift#L125 can also just be moved over to here.

The main reason that I was reluctant to move it in the first place: **I don't know how to test this.** And I know that you have several pieces of strange-looking code in your project that are actually work-arounds and have their reasons ...

Actually, I am wondering how you guys are testing for iOS 12, and very interested to learn more about this :-) . 
I have seen check-in comments in your code like "fix the build for iOS 12.4" - is this configuration available somewhere? Or can it be enabled manually / locally somehow?

TLDR: I am happy to clean this up.

Reply:
iOS 12 is a little tricky and by tricky I mean impossible to test in the Simulator, since Apple never provided a Simulator for 12.5. On our dev side we either had an old iPhone with 12.x laying around that we could upgrade to 12.5 or even had to buy some used devices (: So if we want to see the code running we have to test on a real device. 

Regarding your issue the compiler should throw an error if you use code that is not available in iOS 12.5. That is why we have all those 
`if #available(iOS 13, *) {
 	//Do stuff that is only available on iOS 13 and higher
 } else {
 	// Fallback to iOS 12 APIs
 }`
scattered all over the code. 

So if a am not mistaken it would be enough to just write:
`navigationItem.largeTitleDisplayMode = .never` without the `if #available(iOS 13, *) {`

Reply:
> iOS 12 is a little tricky and by tricky I mean impossible to test in the Simulator, since Apple never provided a Simulator for 12.5. 

Yes, I am aware of that. It's a pity ...
The delta between iOS 12.4 and 12.5 is mainly the Exposure Notification Framework, right? So in theory you could have a configuration for iOS 12.4 with mocks for all layers below the visible user interface (or at least for the ENF), which would allow at least to test the screens and the animations without a true device ... and actually, I was hoping that you have already prepared something similar ... (and I have no idea how much effort it would be to prepare such a configuration ...)

Reply:
> On our dev side we either had an old iPhone with 12.x laying around that we could upgrade to 12.5 or even had to buy some used devices (: So if we want to see the code running we have to test on a real device.

Another thought. Actually, I have a device with iOS 12.5, so in theory I could support the project with testing.
Of course I don't have the entitlements for ENF. But the idea of opening the TestFlight for interested beta testers from the community has been brought up several times (although so far always been rejected ...)
Do you think it would be worth to open again a github ticket on this idea (or to revive the discussion on one of the already available tickets)?